app/ViewModels/Concerns/Wallet/CanForge.php

```diff
@@ -79,4 +79,23 @@ public function isMissing(): bool
             ->filter(fn ($performance) => $performance === false)
             ->count() > 1;
     }
+
+    public function isForged(): bool
```

Comment:
```suggestion
    public function hasForged(): bool
```

Keep those method names in line with the `Slot` class.